### DISCUSSION TITLE
Box Plot Type Error

### DISCUSSION THREAD

**Discussion by PhantomAlex1** [cooperrc/computational-mechanics#244] [General]
I'm doing section 2 of the homework and when doing the box plot I'm getting the error "TypeError: '<' not supported between instances of 'float' and 'str'." The box plot still shows up with all the information and it looks correct is there something I'm missing. This error shows up for the line to calculate the quartiles.
![image_2024-06-06_232252484](https://github.com/cooperrc/computational-mechanics/assets/171092226/47e612a1-603c-47d0-bb29-dd3866a8b433)

**Comment by cooperrc** [comment]
There's a space after `plt.boxplot (`

Is that causing an issue? 

Otherwise, `transistor` might not be a useable numpy array for `percentile` to process.
